### ISSUE TITLE
Calling isValid() after validate() gives the wrong result if a remote validation is in progress

### ISSUE THREAD

**Issue by theirishpenguin** [formvalidation/formvalidation#962]
Example,
```
$('button.save').click(function(e){
  var validator = $(e.target).closest('form').data('bootstrapValidator');
  validator.validate();
  // Remote validate is in progress and not yet completed?
  validator.isValid(); // Gives wrong answer
})
```

Is it possible to pass an option to validate() to make it wait on the remote call?

**Comment by nghuuphuoc** [comment]
Fixed in v0.6.1-dev. 
See https://github.com/formvalidation/formvalidation/issues/1318#issuecomment-70209098

**Comment by rbw0** [comment]
I'm getting null back from validator.isValid() in 0.6.1-dev if there are incomplete remote calls, whereas 0.6.0 returned false. What's the expected behavior here?

Documentation states:
isValid(): Boolean — Returns true if all form fields are valid. Otherwise, returns false.


Wrapping the isValid call in ajaxStop() seems to help off the issue. But that's only if not using a delay (guessing delay uses setTimeout which causes the requests to trigger after ajaxStop())
